test/unit/util_spec.js

```diff
@@ -14,10 +14,37 @@
  */
 
 import {
-  ReadableStream, removeNullCharacters, stringToPDFString
+  bytesToString, ReadableStream, removeNullCharacters, stringToBytes,
+  stringToPDFString
 } from '../../src/shared/util';
 
 describe('util', function() {
+  describe('bytesToString', function() {
+    it('handles non-array arguments', function() {
+      expect(function() {
+        bytesToString(null);
+      }).toThrow(new Error('Invalid argument for bytesToString'));
+    });
+
+    it('handles array arguments', function() {
```

Comment:
Since there's two code-paths in `bytesToString`, if it's not too much trouble could you please add an additional test here covering the length `> 8192` case?

Reply:
Done in the new commit. Good catch!